providers/fab/tests/unit/fab/www/test_auth.py

```diff
@@ -246,3 +246,27 @@ def test_has_access_dag_entities_when_logged_out(self, mock_get_auth_manager, ap
 
         mock_call.assert_not_called()
         assert result.headers["Location"] == "login_url"
+
```

Comment:
It should be in `test_utils`

Reply:
Do you mean moving it to `providers/fab/tests/unit/fab/auth_manager/cli_commands/test_utils.py` ?

Reply:
Yes. The code is in `utils`, the test associated to it should be in `test_utils`

Reply:
I created a new file in `providers/fab/tests/unit/fab/www/test_utils.py` and moved the test there since adding it to cli felt not right. Please let me know if it should be changed.